lib/wickedbase.pm

```diff
@@ -737,6 +738,51 @@ sub record_console_test_result {
     $self->write_resultfile($filename, $content);
 }
 
+sub check_logs {
+    my $self = shift;
+    my @units = qw(wickedd-nanny wickedd-dhcp4 wickedd-dhcp6 wicked wickedd);
+    my $default_exclude = 'wickedd=process \d+ has not exited yet; now doing a blocking waitpid';
+    $default_exclude .= ',wickedd-dhcp6=Link-local IPv6 address is marked duplicate:';
+    $default_exclude .= ',wickedd-nanny=: device has been deleted';
+    $default_exclude .= ',wickedd-dhcp4=unable to confirm lease';
+    $default_exclude .= ',wickedd-nanny=: call to org.opensuse.Network.Interface.waitLinkUp\(\) failed: General failure';
+    $default_exclude .= ',wickedd-nanny=: call to org.opensuse.Network.Interface.waitLinkUp\(\) failed: Object does not support requested method';
+    $default_exclude .= ',wickedd-nanny=: failed to bring up device, still continuing';
+    $default_exclude .= ',wickedd=error retrieving tap attribute from sysfs';
+
+    my $exclude_var = get_var(WICKED_CHECK_LOG_EXCLUDE => $default_exclude);
+    my $exclude_test_var = get_var('WICKED_CHECK_LOG_EXCLUDE_' . $self->{name}, '');
```

Comment:
I am against this variable , I think it will create too much mess . let's start from just single global variable.  Also do you have real examples where you really need it ?  Also in such a case I would prefer to switch to json and still have it as single list but with some additional field which controls whether it should be applied unconditionally or only to some certain cases

Reply:
I would keep it but ok... Because it is a nice way to instantly react/debug on unexpected things..

Reply:
ok

Reply:
will keep it!